### DISCUSSION TITLE
plex_debrid - home assistant

### DISCUSSION THREAD

**Discussion by vednolacni** [itsToggle/plex_debrid#445] [Ideas]
Is there a plan to make plex_debrid as a home assistant add on?

**Comment by itsToggle** [comment]
since plex_debrid is only a middle man between different services, I dont think thats necesarry. There a home assistant implementations for Plex out there, which should allow you to add something to your watchlist. Id have to check, but I think something like that should exist for overseerr aswell. Adding something to your watchlist is all the interaction you need for plex_debrid to download it.

**Reply by vednolacni** [reply]
i was wondering for more like a alternative way to run it.  i have a Home Assistant Operating System on ODROID-N2+. So i was wondering if perhaps i can install it directly on HAOS so that i don't need a docker for example. I know ther is already a plex add on so i would only need to have add on for Mounting realdebrid and add on for plex_debrid to work...

**Comment by ghost** [comment]
I’ve also been looking for this exact method of installation! All of the other required components are already natively available for HA OS. Creating an addon would allow tons more people to use this awesome tool please consider!

**Comment by PhilipGarnero** [comment]
I do agree, I was actually looking into how to make it work with the plex add on already available inside home assistant.
As plex_debrid can run inside docker, I think the setup will be very easy.
We can add a few files and transform your repository into a home assistant add on easily.
The only thing that I don't get is how we should setup the rclone part as it is not inside a docker container.
We might want to add that part of the script to your Dockerfile.
The plex addon for instance has a config file to make the media & the share folders of the home assistant installation available to the plex server : https://github.com/hassio-addons/addon-plex/blob/main/plex/config.yaml#L16
Maybe we could try reusing the instructions [here](https://github.com/itsToggle/plex_debrid/wiki/Setup-Guides#1-mounting-realdebrid-1) so we could package the rclone setup + your script into a single image.
What do you think ?
